src/glmfit.jl

```diff
@@ -1,96 +1,95 @@
 type GlmResp{V<:FPVector,D<:UnivariateDistribution,L<:Link} <: ModResp       # response in a glm model
     y::V                                       # response
     d::D
-    l::L
     devresid::V                                # (squared) deviance residuals
     eta::V                                     # linear predictor
     mu::V                                      # mean response
     mueta::V                                   # derivative of mu w.r.t. eta
     offset::V                                  # offset added to linear predictor (usually 0)
-    var::V                                     # (unweighted) variance at current mu
     wts::V                                     # prior weights
-    wrkwts::V                                  # working weights
+    wrkwt::V                                   # working weights
```

Comment:
Is there a usage of `wrkwts` below?  I thought I had eliminated them all.

Perhaps the `wts` name should be replaced by `weights`.  I think I was too cryptic when I first started writing Julia code.

This still would not resolve `wrkwt` vs `weights` regarding singular or plural.  Most of the names of vectors, `wrkresid`, `wrkwt`, `devresid`, `y`, ... are singular.  For consistency it might be best to make them all singular.